mlflow/models/evaluation/default_evaluator.py

```diff
@@ -768,40 +184,36 @@ def _evaluate_custom_artifacts(custom_artifact_tuple, eval_df, builtin_metrics):
     return artifacts
 
 
-def _compute_df_mode_or_mean(df):
-    """
-    Compute mean (for continuous columns) and compute mode (for other columns) for the
-    input dataframe, return a dict, key is column name, value is the corresponding mode or
-    mean value, this function calls `_is_continuous` to determine whether the
-    column is continuous column.
+# TODO: Move this to the /evaluators directory
+class BuiltInEvaluator(ModelEvaluator):
```

Comment:
Note: This should be at `mlflow/models/evaluation/evaluators/base.py`, but doing so makes git diff even messier. I will do so in a follow-up that gonna be a pure copy-paste.